docs/custom-mao-and-capbm.md

```diff
@@ -94,6 +94,14 @@ Edit `custom-images.json` to have a modified image for the BareMetal case:
 
 ## 6) Now run the MAO
 
+Before running the MAO, we have to undo the change we did earlier by asking the
+cluster-version-operator to manage the machine-api-operator's deployment again.
+Without this, it will not scale the MAO back up.
```

Comment:
but we don't want it to scale the MAO back up, right?  You're running your own outside of the cluster, instead.  If it scales back up, it's not running your custom one.

Reply:
Ok. I misunderstood behavior to be that we were just replacing the image that the CVO was using to scale MAO back up.
I did not see the custom MAO come up either in the default or openshift-machine-api namespaces.

Reply:
Yeah, you won't see it listed as a pod.  The `bin/machine-api-operator start ...` command below is what runs it.  It's running outside the cluster, so you won't see it listed anywhere.